src/test/java/uk/gov/hmcts/reform/sscs/service/ccdupdate/CcdStateUpdateServiceTest.java

```diff
@@ -73,7 +73,8 @@ void shouldSetCcdStateForCancelledHearingsCorrectly(String cancellationReason, S
     @Test
     void shouldSetCcdStateForFailedHearingsCorrectly() {
         // given
-        SscsCaseData caseData = SscsCaseData.builder().state(State.UNKNOWN).ccdCaseId("123").build();
+        SscsCaseData caseData = SscsCaseData.builder()
+            .state(State.UNKNOWN).ccdCaseId("123").build();
```

Comment:
Minor: I meant to new line these separately sorry like..
.state()
.ccdCaseId()
.build